backend/.env.example

```
@@ -64,11 +63,16 @@ SMTP_USER="cbe.system.user@gmail.com"
 COUCHDB_URL=http://couchdb:5984
 COUCHDB_DATABASE=cms-evidence
 COUCHDB_USER=simon
-COUCHDB_PASSWORD=1234
\ No newline at end of file
+COUCHDB_PASSWORD=1234
+
+# Audit-lib OpenSearch Configuration
+AUDIT_PROVIDER=opensearch
+OPENSEARCH_NODE=http://localhost:9200
+OPENSEARCH_USERNAME=admin
+OPENSEARCH_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENSEARCH_PASSWORD key should go before the OPENSEARCH_USERNAME key

(UnorderedKey)